docs/supply-chain.md

```diff
@@ -0,0 +1,172 @@
+
+### How to run the pipeline
+
+#### Using Openshift Web Console
+
+1. Launch the OpenShift Web console.
+2. Select **Pipelines -> Pipelines** from the left hand navigation bar.
+3. Locate the **qtodo-supply-chain** pipeline. It's within the **layered-zero-trust-hub** project.
+4. In the kebab menu (three vertical dots) from the right-hand, select **Start**.
+
+    Review the configurable parameters. Most parameters should be correct with their default values if we are in single-cluster mode. But double-check them just in case.
+
+    At the bottom we have the **workspaces**. These must be configured manually.
+
+    * For **qtodo-source**, select `PersistentVolumeClaim` and the PVC name is `qtodo-workspace-source`.
+    * For **registry-auth-config**, select `Secret` and the name of the secret is `qtodo-registry-auth`.
+
+5. Press **Start** to finish and run the pipeline.
+
+#### Using CLI
+
+We can also start a pipeline execution using a CLI and the Kubernetes API. We start creating a new `PipelineRun` resource referencing the `qtodo-supply-chain` pipeline. Let's create a new file called `qtodo-pipeline.yaml` and copy this content.
+
+```yaml
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  generateName: qtodo-manual-run-
+  namespace: layered-zero-trust-hub
+spec:
+  pipelineRef:
+    name: qtodo-supply-chain
+  taskRunTemplate:
+    serviceAccountName: pipeline
+  timeouts:
+    pipeline: 1h0m0s
+  workspaces:
+  - name: qtodo-source
+    persistentVolumeClaim:
+      claimName: qtodo-workspace-source
+  - name: registry-auth-config
+    secret:
+      secretName: qtodo-registry-auth
+```
+
+Using the previously created definition, we start a new execution of the pipeline using `oc`:
+
+```shell
+oc create -f qtodo-pipeline.yaml
+```
+
+### Pipeline tasks
+
+The pipeline we have prepared has the following steps:
+
+* **qtodo-clone-repository**. Clones the `qtodo` repository.
+* **qtodo-build-artifact**. Builds an _uber-jar_ of `qtodo` application.
+* **qtodo-sign-artifact**. Signs the JAR file generated during the build process.
+* **qtodo-verify-artifact**. Verifies the JAR signature generated in the previous step.
+* **qtodo-build-image**. Builds a container with the `qtodo` application and upload it to an image registry.
+* **qtodo-sign-image**. Signs the container image.
+* **qtodo-generate-sbom**. Generates an SBOM from the image.
+* **qtodo-sbom-attestation**. Creates a (signed) attestation, and attaches it to the image.
+* **qtodo-upload-sbom**. Uploads the generated SBOM file  to RHTPA.
+* **qtodo-verify-image**. Verifies the attestation and the signature attached to the image.
+
+### Inspecting the results
```

Comment:
Are we intending to fill out this section. Some ideas include:

* Viewing the image in Quay
* Seeing the shield associated with the tag illustrating the image is signed
* Seeing the attestation published to Quay
* Viewing items in the Rekor UI
* Viewing the SBOM in the RHTPA uI